Content.Shared/Chemistry/Reagent/ReagentPrototype.cs

```diff
@@ -44,9 +47,15 @@ public sealed class ReagentPrototype : IPrototype, IInheritingPrototype
         [DataField("desc", required: true)]
         public string Description { get; } = default!;
```

Comment:
I'm not a fan of these unlocalised fields remaining public.
I can envision people using the unlocalised ones instead of the localised one, leading to localisation IDs being outputting in places.

Reply:
OOP bad, but I moved them to private.